tests/suites/test_suite_psa_crypto.function

```diff
@@ -1337,6 +1337,34 @@ typedef enum
     INJECT_ANTICIPATE_KEY_DERIVATION_2,
 } ecjpake_injected_failure_t;
 
+#if defined( MBEDTLS_ECP_RESTARTABLE)
+
+static void interruptible_signverify_get_minmax_completes(uint32_t max_ops,
+        psa_status_t expected_status, size_t *min_completes,
+        size_t *max_completes)
+{
+
+    /* This is slightly contrived, but we only really know that with a minimum
+       value of max_ops that a successful operation should take more than one op
+       to complete, and likewise that with a max_ops of
+       PSA_INTERRUPTIBLE_MAX_OPS_UNLIMITED, it should complete in one go. */
+    if( max_ops == 0 || max_ops == 1 )
+    {
+
+        if( expected_status == PSA_SUCCESS )
+            *min_completes = 2;
+        else
+            *min_completes = 1;
+
+        *max_completes = PSA_INTERRUPTIBLE_MAX_OPS_UNLIMITED;
+    } else
```

Comment:
This would be changing it back to the way it was prior to extracting it into the function, which Gilles previously wasn't happy with. Github is hiding these conversations from me, so its difficult to refer back to it here, unfortunately.

Reply:
Yeah, big PRs are intrinsically hard to review, but github is really not making it any easier by hiding stuff like this.

Reply:
I'm guessing you're refering to [this](https://github.com/Mbed-TLS/mbedtls/pull/6737#discussion_r1087879898)?

I was objecting to
```
    if( max_ops == PSA_INTERRUPTIBLE_MAX_OPS_UNLIMITED )
    {
        min_completes = 1;
        max_completes = 1;
    }
    else
    {
        min_completes = 2;
        max_completes = PSA_INTERRUPTIBLE_MAX_OPS_UNLIMITED;
    }
```
Specifically, setting `min_completes = 2` whenever `max_ops` is finite.

I agree with Manuel that `max_completes` should be 1 only when `max_ops` is UNLIMITED.

But now that we have the code in one place, I won't insist on it: it's easy to fix later, and reasonably easy to discover.